md/tomcat-bundle.md

```diff
@@ -35,7 +35,7 @@ The fully qualified folder path (including the BonitaBPM-x.y.z-Tomcat-7.0.67 fol
 * Windows: `C:\BonitaBPM`. 
 * Linux: in `/opt/BonitaBPM`. Make sure that Linux user account used to execute Tomcat is the owner of the folders and files.
 
-::: warning  
+::: warning
 Whatever location you choose, do not leave blank spaces in the path to the directory, nor in the folder name.
```

Comment:
Do we have know issues with blank spaces or is that more a precaution? If it's a known issue I think it would be nice to say that explicitly (to prevent users from trying).